tests/visual/calendar.html

```diff
@@ -1,8 +1,8 @@
 <!doctype html>
-<html>
-
+<html lang="en" class="k-typography">
 <head>
     <title>Calendar</title>
+    <meta cahrset="utf-8" />
```

Comment:
Typo: `cahrset`. Found in a few of the test files.

Reply:
The charset is needed for emoji rendering. I will fix and re-upload